### PR TITLE
Removes a stinky randomly generated AI name

### PR DESCRIPTION
**Pull request by Eagletanker**
## About the PR 

Removes "Fembot" from the list of random AI names. 

## Why's this needed? 

Its kinda gross and probably shouldnt be a name. No changelog likely needed.

### CODE REVIEW

File: _std/__build.dm [hunk 1]
```diff
@@ -109,7 +109,6 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define MAP_OVERRIDE_WRESTLEMAP   // Wrestlemap by Overtone
 // #define MAP_OVERRIDE_POD_WARS   // 500x500 Pod Wars map
 
-
```
Comment (Zonespace27) [inline _std/__build.dm:112]: unsure on why this is being edited

Comment (PrimeNumb) [reply _std/__build.dm:112]: just don't commit it

**Comment by virvatuli-gh** [comment]
Fembot is just a portmanteau of Female Robot and has been a sci-fi trope forever, without being anything really bad. Does this have some new context/was it co-opted into meaning something rude? I don't really see the point in censoring it out of the code otherwise.

**Comment by ZeWaka** [comment]
yeah I don't really get this, it's just a portmanteau like virva said - the only usage i've seen in my life is by queer music artists

**Comment by Azrun** [comment]
> A docile, unthinking and conformist woman.

I was unaware of this every being a thing.

Alternatives include: Gynoid 
But that one feels funny...

**Comment by flrsh** [comment]
Not super familiar with fembot the sci-fi trope, but a quick Google search makes me believe that we're better off not having this as a random AI name. For instance, all the top results in Google images. 

Also yeah please discard your changes to `__build.dm`.

**Comment by UrsulaMejor** [comment]
fix the merge conflicts and I will merge

**Comment by penny64** [comment]
So for context

[Laurel Hubbard & Futurama's Horrifying Trans Olympic Athlete Episode](https://youtu.be/AdYv8a8oCDQ)

Here's what really popularized the modern meaning of that term, if you skip the video the title of the episode "Bender Gets Bent" (eugh) should set the team.


It's really unfortunate to see the administration team and some of the community take a defensive stance like this every time something inappropriate is brought up.

Thankfully no one got roped into a long painful session trying to explain how this victimizes them this time, but it sets a bad tone. 
Don't stick with your own experience if you can't "see the point in censoring it out of [...] the code".
Because number one, yelling "freeze peaches" because something that makes someone feel bad is deeply inflammatory, and two because of the reason above: why would you treat a term possibly being innapropriate so flippantly?

**Comment by flrsh** [comment]
Re: Penny—

Just because you're familiar with a very particular context of a term doesn't mean everyone else is. Virva asked for additional context and if the term was co-opted into something rude. You taking her words out of context and drawing comparisons to those who scream about free speech when confronted about their obnoxious and/or hateful behavior is pretty offensive. 

In the future, I recommend explaining why something is bad to people, rather than acting sanctimonious and belittling them ("taken serious by the people who matter", implying everyone who wasn't aware of the issue didn't matter), when they don't have the same information. It is the kinder thing to do.

I also encourage you to share all times people were "roped into a long painful session trying to explain how this victimizes them" in regards to inappropriate content in an admin feedback thread, as I don't recall instances of that, and it really shouldn't be a thing.

**Comment by ZeWaka** [comment]
> It's really unfortunate to see the administration team and some of the community take a defensive stance like this every time something inappropriate is brought up.

what? like i said before - "the only usage i've seen in my life is by queer music artists" - I've never watched a single episode of futurama, and I only watch the olympics for ping pong. I've only really heard it in a Charli XCX song. The reason I merged this is because i read flourish's comment and looked at the google images page. still had no clue as to what you referred to in your post.

Please don't accuse me of being defensive of victimizing people if I don't even have the knowledge of what is potentially bothering people. i think your ire is being misdirected at the wrong people.

**Pull request merged**